src/features/CourtCaseDetails/Tabs/Panels/Offences/Offence/OffenceMatching.tsx

```diff
@@ -46,7 +46,17 @@ export const OffenceMatching = ({
   const currentUser = useCurrentUser()
 
   const offenceMatchingException = isCaseUnresolved && getOffenceMatchingException(exceptions, offenceIndex)
-  const offenceMatchingExceptionMessage = findExceptions(courtCase, courtCase.aho.Exceptions, ExceptionCode.HO100304)
+  const offenceMatchingExceptionMessage = findExceptions(
+    courtCase,
+    courtCase.aho.Exceptions,
+    ExceptionCode.HO100304,
+    ExceptionCode.HO100328,
+    ExceptionCode.HO100507
+  )
+
+  const doNotDisplayNoPncSequenceBox = exceptions.some((e) =>
+    [ExceptionCode.HO100304, ExceptionCode.HO100328, ExceptionCode.HO100507].includes(e.code)
```

Comment:
A pedantic suggestion: can we rename it to `doNotDisplayPncSequenceBox` instead?